### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by hadeel-farhan**
Created the filter, search, card, and carousel components. I still need to be able to take in an object and make the test object be passed in because its all hardcoded.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,9 +12,12 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.3",
+    "bootstrap-icons": "^1.10.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^2.1.3",
```
Comment (hunterrudnet) [inline package.json]: why the huge downgrade?

File: src/carousel/card.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" />
+    <link href="card.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" rel="stylesheet" />
+    <!-- <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script> -->
```
Comment (hunterrudnet) [inline src/carousel/card.html]: what are these comments?

File: src/carousel/card.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import './card.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap-icons/font/bootstrap-icons.css';
+
+export default class Card extends Component {
```
Comment (hunterrudnet) [inline src/carousel/card.js]: why class based React Components? Let's try to convert these to functional components if possible using ES6 styling!

**Review by hunterrudnet — CHANGES_REQUESTED** [review]
Was this the MUI version? I don't see any MUI in here and lots of it just seems ripped from another repository. Let's try to rework it so that the components are functional, and we can use Material-UI where applicable! Thanks

**Comment by hadeel-farhan** [comment]
Carousel from https://www.npmjs.com/package/react-spring-3d-carousel

File: src/components/Carousel/Cards.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    onChangeInput = (e) => {
+        this.setState({
+            [e.target.name]: parseInt(e.target.value, 10) || 0
+        });
+    };
+
+
+
+    render() {
```
Comment (hunterrudnet) [inline src/components/Carousel/Cards.js:57]: can we please change this to functional?

File: src/components/Carousel/Card.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+.icon {
+    width: 3rem;
+}
+
+.filter {
+    max-width: 40%;
+}
+
+/* .css-1qzevvg {
```
Comment (hunterrudnet) [inline src/components/Carousel/Card.css:22]: can we remove this comment, and rename the class selector to something more readable?

File: src/App.js [hunk 6]
```diff
@@ -1,27 +1,34 @@
...
-            <Route index element={null}/>
-            <Route path="/see-more" element={<SeeMoreList {...props}/>}/>
-            <Route path="/profile" element={<Profile/>}/>
-          </Routes>
-        </div>
-      </BrowserRouter>
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        {/* <LoginButton />
```
Comment (hunterrudnet) [inline src/App.js:14]: Minor, but let's uncomment these to make sure everything works, and add a route for your components to test the render

**Review by hunterrudnet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**